tools/ImporterV2/App.config

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+	<!--
+  This code allows to import resources from a storage container to a FHIR service.
+  Input data must be provided in ndjson format in one container in Azure blobs storage.
+  -->
+	<appSettings>
+		<!-- Azure blob storage parameters -->
+		<add key="ConnectionString" value="{Add your BlobServiceClient connection string here}" />
+		<add key="ContainerName" value="{Add your container}" />
+		<!-- 
+    the type of files it'll search for in the container 
+    do not add a file extension as ndjson is assumed and added to the ResourceType automatically
+    -->
+		<add key="ResourceType" value="Observation" />
+    <!-- 
+    the number we want to import at a time 
+    of course this can be a much larger value than 20 so adjust as needed
+    -->
+		<add key="NumberOfBlobsForImport" value="20" />
+		<add key="FhirEndpoint" value="{add your FHIR endpoint}" />
+		
+		<!-- the following are for getting a token -->
+		<add key="TokenEndpoint" value="{add login url to your oauth2/token endpoint}" />
+		<add key="grant_type" value="Client_Credentials" />
+		<add key="client_id" value="{add your client id}" />
+		<add key="client_secret" value="{add your client secret}" />
+		<add key="resource" value="{add your FHIR endpoint}" />
```

Comment:
Why it is not jus a connection string?

Reply:
It uses the connection string for getting the blob container and lines 25-28 in the app.config for gettign the auth token needed for $import status.

Reply:
Will it work the same way for OSS FHIR?